src/Microsoft.AspNetCore.Blazor.Browser.JS/tsconfig.json

```diff
@@ -11,5 +11,6 @@
   "exclude": [
     "node_modules",
     "wwwroot"
-  ]
+  ],
+  "compileOnSave": true
```

Comment:
Does this really do something useful? Since we're building via Webpack, doing `compileOnSave` doesn't help us, or does it?